proposals/0035-linalg-matrix.md

```diff
@@ -54,131 +54,132 @@ on a base set of functionality for inclusion in the `hlsl` namespace.
 namespace dx {
 namespace linalg {
 
-template <MatrixComponentType ComponentTy, uint M, uint N, MatrixUse Use,
-          MatrixScope Scope>
+template <MatrixComponentEnum ElementType, uint DimA> struct VectorRef {
```

Comment:
Should this be just `ComponentTypeEnum` since we are using this for both Matrix and VectorRef?